model/ravelWrap.cc

```diff
@@ -316,16 +316,17 @@ namespace minsky
   {
     double r=ravelDefaultRadius, z=zoomFactor();
     if (ravel) r=1.1*z*ravel_radius(ravel);
-    ports[0]->moveTo(x()+1.1*r, y());
-    ports[1]->moveTo(x()-1.1*r, y());
+    double s=r*z;    
```

Comment:
Why do you need to square the zoomFactor? It doesn't make sense.

Reply:
Maybe the first r (ravelDefaultRadius) needs to be multiplied by the zoomFactor

Reply:
Thanks, your suggestion did the trick and is much simpler than what I proposed. :)